### ISSUE TITLE
Deploy.ps1 throws parsing errors in VA based off next branch

### ISSUE THREAD

**Issue by pavolum** [microsoft/botframework-solutions#3332]
#### What project is affected?
VA Core on next branch

#### What language is this in?
C#

#### What happens?
Deploy script does not start due to parsing error

#### What are the steps to reproduce this issue?
Pull the latest from next, open VirtualAssistantSample solution, run deploy.ps1, errors will immediately appear

#### Can you share any logs, error output, etc.?
```
PS C:\...\botframework-solutions\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample> .\Deployment\Scripts\deploy.ps1
At C:\...\botframework-solutions\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample\Deployment\Scripts\deploy.ps1:238 char:9
+         | Out-Null
+         ~
An empty pipe element is not allowed.
At C:\...\botframework-solutions\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample\Deployment\Scripts\deploy.ps1:283 char:9
+         | Out-Null
+         ~
An empty pipe element is not allowed.
+ CategoryInfo          : ParserError: (:) [], ParseException
+ FullyQualifiedErrorId : EmptyPipeElement
```

**Comment by pavolum** [comment]
I was running pwsh 6.2.3 
I updated to 7.0.0 and the script works again :)

**Comment by darrenj** [comment]
Addressed in #3366
